submodules/faxbox/faxbox.js

```diff
@@ -632,6 +632,12 @@ define(function(require) {
 					faxboxId: faxboxId
 				},
 				success: function(data) {
+					var custom_smtp_address = _.get(data, 'metadata.custom_smtp_address');
+
+					if (custom_smtp_address) {
+						data.data.custom_smtp_address = custom_smtp_address;
```

Comment:
Because this value is added to the `data` object, it is included in the update data when `faxboxUpdate` is called, although it should not be part of the faxbox data.

If you see a freshly created faxbox from SmartPBX, it does not include this property, but if you open it in the Callflows app, and save it without changing anything, this property will be added to the faxbox document.

Before:
![image](https://github.com/2600hz/monster-ui-callflows/assets/893877/1b9c5376-efd6-40eb-96f4-89b8438eeb46)

After:
![image](https://github.com/2600hz/monster-ui-callflows/assets/893877/8b9d43ce-c64f-4abe-b80f-5cb702ebcadc)